### PR TITLE
feat: Copy Link to a message

### PR DESCRIPTION
**Pull request by pranavrajs**
- Add support for links to a message.

Ref: [CW-1373](https://linear.app/chatwoot/issue/CW-1373/copy-link-to-a-message)

### CODE REVIEW

**Comment by pranavrajs** [comment]
@sojan-official Sure works for me. If someone who has the bandwidth can take this over, that would be helpful.

**Comment by sojan-official** [comment]
@shivamsinghchahar @nithindavid, Could either of you take a look at this?

**Comment by scmmishra** [comment]
@sojan-official I can take a look

**Comment by scmmishra** [comment]
@pranavrajs whenever you have the time, can you help resolve the conflicts in message_finder?

**Comment by nithindavid** [comment]
@pranavrajs @scmmishra Is this ready to test?

**Comment by scmmishra** [comment]
@nithindavid I'll have this ready by tonight

**Comment by pranavrajs** [comment]
@scmmishra This is a good find, let me see if I can find a fix for this.

**Comment by muhsin-k** [comment]
@pranavrajs I have tested this PR. Please see this video
https://www.loom.com/share/410fe0376d454ff1983e6dc6bb78023b

**Comment by pranavrajs** [comment]
@muhsin-k Great feedback. Let me work on it.

**Comment by pranavrajs** [comment]
Note to self: Hide Translate button in the image only messages

File: app/javascript/dashboard/components/widgets/conversation/MessagesView.vue [hunk 1]
```diff
@@ -302,8 +297,16 @@ export default {
...
       this.selectedTweetId = tweetId;
     },
-    onScrollToMessage() {
-      this.$nextTick(() => this.scrollToBottom());
+    onScrollToMessage({ messageId = '' } = {}) {
+      this.$nextTick(() => {
+        const messageElement = document.getElementById('message' + messageId);
+        if (messageElement) {
+          messageElement.scrollIntoView({ behavior: 'smooth' });
+          this.fetchPreviousMessages();
```
Comment (pranavrajs) [inline app/javascript/dashboard/components/widgets/conversation/MessagesView.vue:305]: We will just fetch the immediate next page as well immediately.

**Comment by pranavrajs** [comment]
@scmmishra @muhsin-k I hope everything is ready and all the review comments are addressed. Let me know if you find anything else. 
https://www.loom.com/share/1ec3567ac29e4a4ea0aa09e8b4a8af6a

File: app/javascript/dashboard/components/widgets/conversation/MessagesView.vue (hunk 1 above)
Comment (scmmishra) [reply app/javascript/dashboard/components/widgets/conversation/MessagesView.vue:305]: Sounds good

**Review by scmmishra — APPROVED** [review]

**Review by muhsin-k — APPROVED** [review]
@pranavrajs Please resolve the conflicts.

**Pull request merged**